go.mod

```diff
@@ -66,3 +66,5 @@ require (
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
 )
+
+replace github.com/wal-g/storages => github.com/slesarev-hub/storages v0.0.0-20210616180710-f709c4ac97aa
```

Comment:
I understand this is a WIP PR but make sure you remove this before the final review. Changes/updates to the storages module should be merged there first.